rebar.config.script

```diff
@@ -32,7 +32,15 @@ CouchJSName = case os:type() of
         "couchjs"
 end,
 CouchJSPath = filename:join(["priv", CouchJSName]),
-Version = string:strip(os:cmd("git describe --always"), right, $\n),
+Version0 = string:strip(os:cmd("git describe --always"), right, $\n),
+% When we are not building from a git checkout, the above will fail.
+% If that is the case, we use the COUCHDB_VERSION variable that has
+% been set by the top level Makefile
+Version = case Version0 of
+    "fatal:" ++ _Msg ->
```

Comment:
> Hm...but how we'll transfer COUCHDB_VERSION value to the tarball in this case?

It looks like I got fed up with trying to solve this programmatically, so I made this: https://github.com/apache/couchdb/blob/master/version.mk — e.g.: we have to maintain that file by hand. like we do with `acinclude.m4` in 1.x.x: https://github.com/apache/couchdb/blob/1.x.x/acinclude.m4.in#L19-L21